docs/using-cinder-csi-plugin.md

```diff
@@ -191,33 +191,56 @@ Following prerequisite needed for volume snapshot feature to work.
 1. Enable `--feature-gates=VolumeSnapshotDataSource=true` in kube-apiserver
 2. Make sure, your csi deployment contains external-snapshotter sidecar container, external-snapshotter sidecar container will create three crd's for snapshot management VolumeSnapshot,VolumeSnapshotContent, and VolumeSnapshotClass. external-snapshotter is a part of `csi-cinder-controllerplugin`
 
-For Snapshot Creation and Volume Restore, please follow  below steps:
+For Snapshot Creation and Volume Restore, please follow below steps:
 
 * Create Storage Class, Snapshot Class and PVC    
 ```
 $ kubectl -f examples/cinder-csi-plugin/snapshot/example.yaml create
-```     
+storageclass.storage.k8s.io/csi-sc-cinderplugin created
+volumesnapshotclass.snapshot.storage.k8s.io/csi-cinder-snapclass created
+persistentvolumeclaim/pvc-snapshot-demo created
+``
```

Comment:
there seems to be issue in formatting of doc ., may be because of missing quote above

Reply:
yes, will update, thanks